app.js

```diff
@@ -1,12 +1,18 @@
 const express = require("express");
 const { runServerCheck } = require("./controllers/api.controllers");
-const { getTopics } = require("./controllers/topics.controllers");
+const {
+	getTopics,
+	getAllEndpoints,
+} = require("./controllers/topics.controllers");
 const { serverError, customError } = require("./errors");
 const app = express();
+app.use(express.json());
```

Comment:
we don't need to use express.json() until doing a patch or post request (handling an incoming request body) - just something to bear in mind going forwards that it's best practice to only include code that is necessary for the ticket you're working on 🧠